src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/AuthenticationHelper.Digest.cs

```diff
@@ -407,9 +406,10 @@ private void Parse(string challenge)
                         break;
 
                     // Ensure value is valid.
-                    // Opaque and Domain can have empty string
+                    // Opaque, Domain and Realm can have empty string
                     if (value == string.Empty &&
-                       (!key.Equals(Opaque, StringComparison.OrdinalIgnoreCase) && !key.Equals(Domain, StringComparison.OrdinalIgnoreCase) &&
+                       (!key.Equals(Opaque, StringComparison.OrdinalIgnoreCase) &&
+                        !key.Equals(Domain, StringComparison.OrdinalIgnoreCase) &&
                         !key.Equals(Realm, StringComparison.OrdinalIgnoreCase)))
```

Comment:
```suggestion
                        (!key.Equals(Opaque, StringComparison.OrdinalIgnoreCase) &&
                         !key.Equals(Domain, StringComparison.OrdinalIgnoreCase) &&
                         !key.Equals(Realm, StringComparison.OrdinalIgnoreCase)))
```
Nit: Needs indentation fixed ;)

Reply:
Though, I would rather drop the useless parenthesis pair ...